cmd/datamon/cmd/cli_test.go

```diff
@@ -493,12 +493,15 @@ func listBundles(t *testing.T, repoName string) (bundleListEntries, error) {
 		}
 		rle := bundleListEntry{
 			rawLine: line,
-			hash:    strings.TrimSpace(sl[0]),
+			hash:    strings.TrimSpace(sl[0]), // bundle key ID
 			message: strings.TrimSpace(sl[2]),
 			time:    t,
 		}
 		bles = append(bles, rle)
 	}
+	// bundles are ordered by lexicographic order of keys (ksuids).
```

Comment:
Drop ksuid here it is an internal implementation of the core package.
Also, use Bundle ID

Reply:
ok